tests/ngff/test_ngff.py

```diff
@@ -485,6 +485,82 @@ def test_set_transform_image(ch_shape_dtype, arr_name):
         ]
 
 
+@pytest.mark.parametrize(
+    "transforms",
+    [
+        (
+            [TransformationMeta(type="identity")],
+            TransformationMeta(type="scale", scale=(1.0, 1.0, 1.0, 1.0, 1.0)),
+            TransformationMeta(
+                type="translation", translation=(0.0, 0.0, 0.0, 0.0, 0.0)
+            ),
+        ),
+        (
+            [
+                TransformationMeta(
+                    type="scale", scale=(1.0, 2.0, 3.0, 4.0, 5.0)
+                )
+            ],
+            TransformationMeta(type="scale", scale=(1.0, 2.0, 3.0, 4.0, 5.0)),
+            TransformationMeta(
+                type="translation", translation=(0.0, 0.0, 0.0, 0.0, 0.0)
+            ),
+        ),
+        (
+            [
+                TransformationMeta(
+                    type="translation", translation=(1.0, 2.0, 3.0, 4.0, 5.0)
+                )
+            ],
+            TransformationMeta(type="scale", scale=(1.0, 1.0, 1.0, 1.0, 1.0)),
+            TransformationMeta(
+                type="translation", translation=(1.0, 2.0, 3.0, 4.0, 5.0)
+            ),
+        ),
+        (
+            [
+                TransformationMeta(
+                    type="scale", scale=(2.0, 2.0, 2.0, 2.0, 2.0)
+                ),
+                TransformationMeta(
+                    type="translation", translation=(1.0, 1.0, 1.0, 1.0, 1.0)
+                ),
+            ],
+            TransformationMeta(type="scale", scale=(2.0, 2.0, 2.0, 2.0, 2.0)),
+            TransformationMeta(
+                type="translation", translation=(1.0, 1.0, 1.0, 1.0, 1.0)
+            ),
+        ),
+    ],
+)
+@given(
+    ch_shape_dtype=_channels_and_random_5d_shape_and_dtype(),
+    arr_name=short_alpha_numeric,
+)
+def test_get_transform_image(transforms, ch_shape_dtype, arr_name):
+    """Test `iohub.ngff.Position.set_transform()`"""
```

Comment:
Test needs to be updated.

Reply:
In the spirit of unit testing, can this be separate for the two methods? Also the docstring needs to be updated.